### PR TITLE
Fixes #17681: Upgrade to 6.1 fails on httpd start beacause of old ncf conf is still present on rpm

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/17681

### CODE REVIEW

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 1]
```diff
@@ -281,6 +281,10 @@ then
   sed -i 's%APACHE_MODULES="${APACHE_MODULES} rewrite dav dav_fs proxy proxy_http.*%# This sources the Rudder needed by Rudder\n. /etc/sysconfig/rudder-webapp-apache%' /etc/sysconfig/apache2
 fi
 
+# Remove old ncf-api-virtualenv conf, or else it will prevent apache start 
+if [ -f /etc/%{apache_vhost_dir}/ncf-api-virtualenv.conf ] && rm /etc/%{apache_vhost_dir}/ncf-api-virtualenv.conf
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec:285]:
```suggestion
[ -f /etc/%{apache_vhost_dir}/ncf-api-virtualenv.conf ] && rm /etc/%{apache_vhost_dir}/ncf-api-virtualenv.conf
```

Comment (amousset) [reply rudder-webapp/SPECS/rudder-webapp.spec:285]: but `rm -f` is probably better anyway

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 2]
```diff
@@ -281,6 +281,10 @@ then
   sed -i 's%APACHE_MODULES="${APACHE_MODULES} rewrite dav dav_fs proxy proxy_http.*%# This sources the Rudder needed by Rudder\n. /etc/sysconfig/rudder-webapp-apache%' /etc/sysconfig/apache2
 fi
 
+# Remove old ncf-api-virtualenv conf, or else it will prevent apache start 
+[ -f /etc/%{apache_vhost_dir}/ncf-api-virtualenv.conf ] && rm -f /etc/%{apache_vhost_dir}/ncf-api-virtualenv.conf
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec:285]:
```suggestion
rm -f /etc/%{apache_vhost_dir}/ncf-api-virtualenv.conf
```

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**